content/blog/advent-of-code.md

```diff
@@ -0,0 +1,70 @@
+---
+{
+  "type": "blog",
+  "author": "Felienne Hermans",
+  "title": "Advent of Code Reading",
+  "description": "Advent of Code Reading",
+  "image": "images/article-covers/Advent-of-Code-Dec-2.png",
+  "draft": false,
+  "published": "2021-11-03",
+}
+---
+
+
+
+## A culture of writing code
+
+In my book and in many talks I have discussed the culture we have in programming (and programming education) where we assume students but also professional learn programming by doing a lot of programming. This is embodied by for example the idea that you "need" an open source project outside of regular hours if you want to be a good programmer, the idea that to learn a language building something small like a todo list app is the best way and also a bit by Advent of Code, where people solve a little programming problem every day in December as a means of deliberate practice.
+
+Now of course there is nothing wrong with writing code, and there is nothing wrong with Advent of code! It is just that only writing code is a bit limited to only write code as a means of getting better. Especially if we know that professionals spend about 60% of their time writing code!
+
+To compare with physicial exercise: if you train for a marathon, you will do various things! Slow low runs, quick interval trainings, core exercisese etc. The goal of code reading clubs has always been to widen the activities we do as a field to advance our own skills.
```

Comment:
```suggestion
To compare with physical exercise: if you train for a marathon, you will do various things! Slow low runs, quick interval trainings, core exercises etc. The goal of code reading clubs has always been to widen the activities we do as a field to advance our own skills.
```

Reply:
Oh. @Nikomus I already released, do you fancy fixing that?